api/composer.json

```diff
@@ -8,6 +8,7 @@
     "craftcms/google-cloud": "^1.4",
     "craftcms/redactor": "2.8.5",
     "jamesedmonston/graphql-authentication": "^1.12",
+    "rosas/universal-dam-integrator": "dev-main",
```

Comment:
Sorry I didn't mention this during the call earlier, but I went ahead and created a release, can you change this to "v0.0.1-alpha"?

https://github.com/lsst-epo/craft3-universal-dam-integrator/releases/tag/v0.0.1-alpha